docs/dev/dev-env.md

```diff
@@ -38,7 +38,11 @@ The development environment is composed of 4 parts:
 
 Run the Docker compose file with only required services to bring up database, message broker and other required system
 ```bash
-docker-compose -f docker-compose.yml -f dev.docker-compose.yml up db phpmyadmin rabbitmq redis-cache
+docker-compose -f docker-compose.yml  -f mysql.docker-composer.yaml -f dev.docker-compose.yml up db phpmyadmin rabbitmq redis-cache
```

Comment:
this approach will break the default behavior where we have only one compose. In the doc, new user can start Squest from a simple "docker-compose up".
So it would be better to add a side compose for postgres.

Reply:
I've fixed it for mysql, but leaved it as is for psql as it is not the default behavior.